extensions/ql-vscode/src/extension.ts

```diff
@@ -951,7 +951,14 @@ async function activateWithInstalledDistribution(
       variantAnalysisSummary: VariantAnalysisApiResponse,
       token: CancellationToken
     ) => {
-      await variantAnalysisManager.autoDownloadVariantAnalysisResult(scannedRepo, variantAnalysisSummary, token);
+
+      const input = [
+        variantAnalysisManager.queue.add(async () => {
+          await variantAnalysisManager.autoDownloadVariantAnalysisResult(scannedRepo, variantAnalysisSummary, token);
+        })
+      ];
+
+      await Promise.all(input);
```

Comment:
I think you should be able to await the call to `queue.add` instead of wrapping it in an array and using `Promise.all`.